### DISCUSSION TITLE
How to handle issues in web components that are fixed in newer versions?

### DISCUSSION THREAD

**Discussion by tlmii** [microsoft/fluentui-blazor#863] [Q&A] [answered]
I'll give the example first:

`<FluentDivider Role="DividierRole.Presentation" Orientation="Orientation.Vertical" />`

This will get flagged by Accessibility Insights because it generates this:

`<fluent-divider role="presentation" orientation="vertical" aria-orientation="vertical"></fluent-divider>`

and you're not allowed to have aria-orientation attributes when role is presentation.

This has been fixed in FAST in https://github.com/microsoft/fast/pull/6771 a few months ago. Will this flow to us? Or will it not because it'll only go into 3.x which we're not using yet? I can't really tell for sure.

In either case, should I file an issue on this repo to track it?

**Comment by dvoituron** [comment, marked as answer]
Hi,
Thanks for your feedback.
We have to fix many of our components to be Accessibility compliant (not yet done).
Yes, could you please create an issue? It will be easier to track.
